docs/UserGuide.md

```diff
@@ -223,77 +491,13 @@ Format: `settier TIER_NUM POINT_THRESHOLD`
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
+  than tier 2.
 
-Examples:
-*  `editc 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `editc 2 ct/ind n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+### Orders
 
-### Setting a customer's note : `setnotec`
-
-Sets a customer's note.
-
-Format: `setnotec INDEX nt/NOTE`
-
-* Sets the note of the customer at the specified `INDEX` to `NOTE`.
-* Customers, by default, have a blank note.
-* The index refers to the index number shown in the displayed customer list.
-* The index **must be a positive integer** 1, 2, 3, …​
-* The note refers to the customer's new note.
-* Any existing note will be overwritten.
-* A customer's note can be removed by setting an empty note.
-
-Examples:
-* `listc` followed by `setnotec 2 nt/Very friendly!` sets the 2nd customer's note as "Very friendly!".
-* `listc` followed by `setnotec 2 nt/` removes the 2nd customer's note.
-* `findc Betsy` followed by `setnotec 1 nt/Vegetarian` sets the 1st customer's note as "Vegetarian" in the results of the `findc` command.
-
-### Appending a customer's note : `appendnotec`
-
-Adds more text to a customer's note.
-
-Format: `appendnotec INDEX nt/NOTE`
-
-* Adds `NOTE` to any existing note of the customer at the specified `INDEX`.
-* The index refers to the index number shown in the displayed customer list.
-* The index **must be a positive integer** 1, 2, 3, …​
-* The note refers to the new text that will be added to the end of the customer's existing note.
-
-Examples:
-* `listc` followed by `appendnotec 2 nt/Very friendly!` will add "Very friendly!" to the 2nd customer's existing note.
-* `findc Betsy` followed by `appendnotec 1 nt/Vegetarian` adds "Vegetarian" to the note of the 1st customer in the results of the `findc` command.
-
-### Locating persons by name: `findc`
-
-Finds persons whose names contain any of the given keywords.
-
-Format: `findc KEYWORD [MORE_KEYWORDS]`
-
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
-
-Examples:
-* `findc John` returns `john` and `John Doe`
-* `findc alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'findc alex david'](images/findAlexDavidResult.png)
-
-### Adding an order : `addo`
+#### Adding an order : `addo`
 
 Adds an order tagged to a customer, to the list of orders.
 
```

Comment:
for line 508, the way to describe index is different from how index has been described beforehand, where it is usually like
![image](https://user-images.githubusercontent.com/88615976/228870319-39c1b7c8-bdd8-4c06-ba79-43d7779d65de.png)